scss/tech.scss

```diff
@@ -0,0 +1,47 @@
+//@import 'colors';
+
+.tech {
+  width: 200px;
+  height: 300px;
+  padding: 0px 25px;
+  margin: 20px;
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background-color: #BED23C;
+  border-radius: 10px;
+
+  &::after {
+    content: ' ';
+    width: 200px;
```

Comment:
Since this width and height are the same as the tech card, you can define them as constants and use them, or define a mixin
https://sass-lang.com/documentation/at-rules/mixin